pkg/zonemgr/zone_file_test.go

```diff
@@ -0,0 +1,56 @@
+package zonemgr
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"os"
+)
+
+var _ = Describe("disk zone file maintenance", func() {
+
+	const (
+		zoneFileName           = "zones/db.vm"
+		zoneFileContent        = "zone file content"
+		zoneFileUpdatedContent = "zone file updated content"
+	)
+	var zoneFile *ZoneFile
+
+	BeforeEach(func() {
+		zoneFile = NewZoneFile(zoneFileName)
+		Expect(os.Mkdir("zones", 0777)).To(Succeed())
+	})
+	AfterEach(func() {
+		Expect(os.RemoveAll("zones")).To(Succeed())
+	})
+
+	testFileFunc := func(fileContent string) {
```

Comment:
I would change `fileContent` to `expectedFileContent`

Reply:
done